params.go

```diff
@@ -124,6 +124,10 @@ type Params struct {
 	// Account is deprecated form of StripeAccount that will do the same thing.
 	// Please use StripeAccount instead.
 	Account string
+
+	// This may be used to provide a new header or override an existing one on
+	// the HTTP request.
+	Headers map[string]string
```

Comment:
A couple fair minor comments here:

1. One of the core values of the Go language is that the standard library's types/functions/conventions should be re-used as often as possible. Instead of defining your own map here, can you make this a `Header` type [from `net/http`](https://godoc.org/net/http#Header)? It's basically just an alias for what you have already, but allows for each key to hold more than one value, which is more correct than what you have here because HTTP allows you to repeat headers multiple times.
2. `Header` also exports an `Add` function, so lets drop your `AddHeader` implementation below, and just prefer the use of that.
3. Convention for Go docstrings is to start out with the name of the thing that they're documenting (e.g. `// Headers may be used to provide new headers or override existing ones on HTTP requests made from stripe-go.`). We don't have a linter turned on for the project the project right now (because there's so many existing problems), but golint would've called this out as a violation. Can you reword accordingly?

Thanks!